bucket/butterflow-ui.json

```diff
@@ -0,0 +1,32 @@
+{
+    "version": "1.1.0",
+    "description": "Graphical user interface for butterflow, a video manipulation tool.",
+    "homepage": "https://github.com/wagesj45/butterflow-ui",
+    "license": "MIT",
+    "notes": "butterflow is packaged with butterflow-ui and does not need to be installed separately.",
```

Comment:
I think this is obsolete for end user.

Reply:
Perhaps the butterflow packaged with butterflow-ui shouldn't be shimmed so butterflow can be updated independently if needed.

Reply:
butterflow is no longer shimmed.